src/doc/man/cargo-metadata.md

```diff
@@ -331,6 +331,11 @@ The JSON output has the following format:
 }
 ````
 
+Notes:
+- For `"id"` field syntax, see [Package ID Specifications] in the reference.
+
+[Package ID Specifications]: ../reference/pkgid-spec.md
```

Comment:
I think it was noted in the meeting that this generates a broken link in the `.txt` file. mdman does not do the same `.md` to `.html` translation that mdbook does, so all the links here should be `.html`.

```suggestion
[Package ID Specifications]: ../reference/pkgid-spec.html
```